src/main/scala/djinni/Main.scala

```diff
@@ -108,13 +111,14 @@ object Main {
 
       override def showUsageOnError = false
       head("djinni generator version", Main.getClass.getPackage.getImplementationVersion)
+      note("General")
       help("help")
       version("version")
       opt[File]("idl").valueName("<in-file>").required().foreach(idlFile = _)
         .text("The IDL file with the type definitions, typically with extension \".djinni\".")
       opt[String]("idl-include-path").valueName("<path> ...").optional().unbounded().foreach(x => idlIncludePaths = idlIncludePaths :+ x)
         .text("An include path to search for Djinni @import directives. Can specify multiple paths.")
-      note("")
+      note("\nJava")
```

Comment:
I've added separators [like in the documentation](https://djinni.xlcpp.dev/djinni-generator/cli-usage/). Not related to your PR in any way but I think it makes the help more readable with it's ever growing number of configuration options.